infra/iam_for_stack_ttl.yaml

```diff
@@ -0,0 +1,54 @@
+# provided by AWS quickstart - but not available hosted on s3, so reproduced here from
```

Comment:
**Question**: This file is not referenced in the main `stack.yaml` or anywhere else. Is that expected? Would like to know how it works.